practice_projects/#Homework pt.py

```diff
@@ -0,0 +1,19 @@
+#Homework pt.1
+
+friend = ("Software is my friend!")
+print (friend)
+
+#Homework pt.2
```

Comment:
I don't see #2: Develop a code that has two variables. One variable is equal to “Software is”, and another variable equal to “my friend”. Concatenate both strings, and print it out as a complete sentence.

Reply:
And missed this one. It's on this push